Automattic/jetpack
# Premium content block: Remove a "0" character after the block upgrade nudge

Fixes #22636

#### Changes proposed in this Pull Request:

* Cast selectedPlanId to boolean to remove a "0" character after block upgrade nudge 

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* WPCOM: Apply D74504-code on your sandbox
* Atomic: Use Jetpack Beta tester and switch to this branch.
* Use a free plan site and add a new Premium content block
* Notice that after the upgrade nudge there's no "0" character.
